js/decklist/decklist.js

```diff
@@ -76,7 +78,7 @@ var Decklist = {
           }
 
           let result = re.exec(line),
-            quantity = parseInt(result[1]),
+            quantity = parseInt(result[1]) ? parseInt(result[1]) : 1,
```

Comment:
This can be simplified to `a || b` as follows:
```suggestion
            quantity = parseInt(result[1]) || 1,
```